src/store/actionTypes.js

```diff
@@ -1,6 +1,13 @@
 export const GET_MOVIES_PENDING = 'GET_MOVIES_PENDING';
 export const GET_MOVIES_SUCCESS = 'GET_MOVIES_SUCCESS';
 export const GET_MOVIES_ERROR = 'GET_MOVIES_ERROR';
+
 export const GET_MOVIE_PENDING = 'GET_MOVIE_PENDING';
 export const GET_MOVIE_SUCCESS = 'GET_MOVIE_SUCCESS';
 export const GET_MOVIE_ERROR = 'GET_MOVIE_ERROR';
+
+export const GET_GENRES_BY_ID_PENDING = 'GET_GENRES_BY_ID_PENDING';
```

Comment:
please, rename this action to GET_GENRES_BY_`IDS`_PENDING, also add `SUCCESS` and `ERROR` actions.